### PR TITLE
Directory based config

### PR DESCRIPTION
**Pull request by 10zinten**
Questions
- [ ] no file starts with lexica https://github.com/Esukhia/botok/blob/b5cd49c2e7e6ccff2ef125d65d0c202a658de253/botok/tries/trie.py#L71
- [ ] Where is `particles.tsv` and other data in resources directory ?

### CODE REVIEW

File: botok/config.py [hunk 1]
```diff
@@ -51,104 +51,19 @@ class Config:
...
-            for el in adjs.glob("*.tsv"):
-                el = Path(__file__).parent / "resources" / Path(el)
-                paths.append(el)
+    def __init__(
+        self, d_pack_path=Path().home() / "Documents" / "pybo" / "bo_general_pack"
+    ):
+        """Create config from dialect_pack path."""
+        self.d_pack_path = d_pack_path
+        self.main = self._get_tok_data_paths("dictionary")
+        self.adjustment = self._get_tok_data_paths("adjustment")
```
Comment (10zinten) [inline botok/config.py:60]:
```suggestion
    def __init__(
        self, d_pack_path=Path().home() / "Documents" / "pybo" / "bo_general_pack"
    ):
        """Create config from dialect_pack path."""
        self.d_pack_path = d_pack
        self.main = self._get_tok_data_paths("dictionary")
        self.adjustment = self._get_tok_data_paths("adjustment")
```

File: botok/config.py [hunk 2]
```diff
@@ -51,104 +51,19 @@ class Config:
...
+
+    def _get_tok_data_paths(self, d_type):
+        data_paths = defaultdict(list)
```
Comment (10zinten) [inline botok/config.py:63]:
```suggestion
        data_path = defaultdict(list)
```

**Comment by ngawangtrinley** [comment]
Lexica was replaced by words

**Comment by 10zinten** [comment]
@ngawangtrinley I have put all the dialect packs like [this](https://github.com/Esukhia/botok-data/tree/master/dialect_packs) and release each dialect like [that](https://github.com/Esukhia/botok-data/releases/tag/v0.0.1)

Then we just have to pass the dialect name like so https://github.com/Esukhia/botok/blob/709b6c4a78b5a67fb60148a0c31ad94c88b25e80/tests/test_config.py#L38 and config will automatically download the lastest release of the dialect pack if not downloaded and initialize the config.

Config can also optionally accept directory path of the dialect pack https://github.com/Esukhia/botok/blob/709b6c4a78b5a67fb60148a0c31ad94c88b25e80/tests/test_config.py#L54 and initialize the config.=

How is it ?

**Pull request merged**